powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/CreateTokenRequest.java

```diff
@@ -30,16 +33,42 @@
 @Data
 public class CreateTokenRequest {
 
+    @Schema(description = "Activation identifier")
+    @NotBlank(message = "Activation ID must not be empty when creating token")
     private String activationId;
+
+    @Schema(description = "Application key")
+    @NotBlank(message = "Application key must not be empty when creating token")
     private String applicationKey;
+
+    @Schema(description = "Identifier of the temporary key for encryption")
     private String temporaryKeyId;
+
+    @Schema(description = "Ephemeral public key used in encryption")
+    @NotBlank(message = "Ephemeral public key must not be empty when creating token")
     private String ephemeralPublicKey;
+
+    @Schema(description = "Encrypted data used in encryption")
```

Comment:
The second part of the sentence doesn't sound right.
```suggestion
    @Schema(description = "Encrypted data")
```